recipes/default.rb

```diff
@@ -5,65 +5,11 @@
 # Copyright (c) 2014 The Authors, All Rights Reserved.
 
 # Configure Supermarket server hostname in /etc/hosts if it isn't there (AWS)
```

Comment:
Maybe replace the comment here with something about how the `supermarket_server` resource below is a compatibility feature for folks still using the default recipe directly. :grin: